studio/components/to-be-cleaned/TooltipListener.tsx

```diff
@@ -1,80 +0,0 @@
-import { useRef, useEffect } from 'react'
```

Comment:
Why is this file being removed?

Reply:
It's only usage is in the `deprecated.ts` file to re-export it, but it isn't used anywhere outside of that.  It made sense to just remove it, but if there is possible future use then it makes sense to keep it.

Reply:
For now lets keep it.

Reply:
Reverted the commit.